README.md

```diff
@@ -57,6 +57,49 @@ Use the following syntax for defining localizations in a JSON file:
 }
 ```
 
+### Inside Leaf templates
+
+When using [Leaf](https://github.com/vapor/leaf) as templating engine, you can add a custom tag for localization inside the templates.
+
+```swift
+public final class LocalizeTag: TagRenderer {
+    public init() {}
+
+    public func render(tag: TagContext) throws -> EventLoopFuture<TemplateData> {
+        try tag.requireParameterCount(1)
+        let lingo = try tag.container.make(Lingo.self)
+
+        let locale = (tag.container as? Request)?.locale ?? lingo.defaultLocale
+        
+        guard let key = tag.parameters[0].string else {
+            throw Abort(.internalServerError, reason: "First parameter for localize tag is no string")
+        }
+        
+        if let body = tag.body {
+            return tag.serializer.serialize(ast: body).map(to: TemplateData.self) { body in
+                guard let rawBodyData = String(data: body.data, encoding: .utf8),
+                    let bodyData = "{\(rawBodyData)}".data(using: .utf8),
+                    let interpolations = try? JSONDecoder().decode([String:String].self, from: bodyData) else {
```

Comment:
The `Lingo` interpolations are actually `[String: Any]`, you can pass integers as well which are then used for pluralizations.

Reply:
Alright, now it supports integers as well.